### PR TITLE
add vert.x web sessions

### PR DESCRIPTION
**Pull request by karesti**
add vert.x web sessions link

### CODE REVIEW

File: _includes/documentation-links-band.html [hunk 1]
```diff
@@ -119,7 +119,10 @@ <h4>Community Projects</h4>
           <a href="https://quarkus.io/guides/infinispan-client" target="_blank">Quarkus</a>
         </li>
         <li>
-          <a href="https://vertx.io/docs/vertx-infinispan/java/" target="_blank">Eclipse Vert.x</a>
+          <a href="https://vertx.io/docs/vertx-infinispan/java/" target="_blank">Eclipse Vert.x - Cluster Manager</a>
```
Comment (oraNod) [inline _includes/documentation-links-band.html:122]:
```suggestion
          <a href="https://vertx.io/docs/vertx-infinispan/java/" target="_blank">Vert.x Cluster Manager</a>
```

File: _includes/documentation-links-band.html [hunk 2]
```diff
@@ -119,7 +119,10 @@ <h4>Community Projects</h4>
...
+        </li>
+        <li>
+          <a href="https://how-to.vertx.io/web-session-infinispan-howto/" target="_blank">Eclipse Vert.x - Web Sessions</a>
```
Comment (oraNod) [inline _includes/documentation-links-band.html:125]:
```suggestion
          <a href="https://how-to.vertx.io/web-session-infinispan-howto/" target="_blank">Vert.x Web Sessions</a>
```

**Review by oraNod — APPROVED** [review]
Hey @karesti Sorry I missed this before I went on PTO. Couple small nits here. I don't think we need "Eclipse" tbh and the hyphen is somewhat superfluous.

**Comment by karesti** [comment]
@oraNod rebased and commits squashed

**Comment by oraNod** [comment]
thanks @karesti !

**Pull request merged**